README.md

```diff
@@ -0,0 +1,111 @@
+| `roundcube_install_version`          | :heavy_multiplication_x:       |       `1.3.3`      | Version to install                                                                                                                                                                             |
+| `roundcube_user`              | :heavy_multiplication_x: | `www-data` | Name of the user to be used for roundcube                                                                                                                                                                |
+| `roundcube_group`                 | :heavy_multiplication_x: | `www-data` | Group to be used for roundcube                                                                                                                                                           |
+| `roundcube_validate_certs`           | :heavy_multiplication_x: | `true`      | Should roundcube validate certs during connection to the mail server |
+| `roundcube_sql_host`                | :heavy_multiplication_x: |        `localhost`      | Host of for the database                                                                                                                                                                |
+| `roundcube_sql_user`             | :heavy_multiplication_x: |        `roundcube`      | Database user                                                                                                                                                                     |
+| `roundcube_sql_database`               | :heavy_multiplication_x: |              | Database name                                                                                                                                                               |
+| `roundcube_default_host`  | :heavy_check_mark:       |     `ssl://{{ mailserver_fqdn_hostname | default('localhost') }}`         | Imap server                                                                                                                                                            |
+| `roundcube_default_port`     | :heavy_check_mark:       |       `993`       | Imap port                                                                                                                                                            |
+| `roundcube_smtp_server`     | :heavy_check_mark:       |       `993`       | smtp server                                                                                                                                                          |
```

Comment:
I think you switched the default for smtp port and server

Reply:
fixed